src/components/JUPTable/JUPTable.tsx

```diff
@@ -229,15 +237,19 @@ const JUPTable: React.FC<IJUPTableProps> = ({ headCells, rows, title, path, Disp
           </TableBody>
         </Table>
       </TableContainer>
-      <TablePagination
-        rowsPerPageOptions={TableRowsPerPageOptions}
-        component="div"
-        count={rows?.length || 0}
-        rowsPerPage={rowsPerPage}
-        page={page}
-        onPageChange={handleChangePage}
-        onRowsPerPageChange={handleChangeRowsPerPage}
-      />
+      {isPaginated ? (
```

Comment:
depending on if these tables will get a lot of updates to them, perhaps memoizing the pagination is a nice little performance boost?

Reply:
memoized